From db8d64b9c34ee827dffe0ea4d20c5ad20b1a02ef Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 19 Jul 2024 15:48:29 +0200
Subject: [PATCH 1064/1695] dts: stm32: remove activation of TAMP and RTC nodes
 at board level

Remove unnecessary activation of TAMP and RTC nodes at board level.
It is already done at SoC level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git df349fef2e9da7e6f8f7919b77906e8e0efbd53a dts: stm32: remove activation of TAMP and RTC nodes at board level]
Change-Id: I78eb805fde1923cec1676767fbc05745e7c47884
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/394366
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts         | 5 -----
 core/arch/arm/dts/stm32mp157a-ed1.dts        | 4 ----
 core/arch/arm/dts/stm32mp157c-ed1.dts        | 4 ----
 core/arch/arm/dts/stm32mp157d-ed1.dts        | 4 ----
 core/arch/arm/dts/stm32mp157f-ed1.dts        | 4 ----
 core/arch/arm/dts/stm32mp15xx-dhcom-som.dtsi | 4 ----
 core/arch/arm/dts/stm32mp15xx-dhcor-som.dtsi | 4 ----
 core/arch/arm/dts/stm32mp15xx-dkx.dtsi       | 4 ----
 core/arch/arm/dts/stm32mp235f-dk.dts         | 5 -----
 core/arch/arm/dts/stm32mp257f-dk.dts         | 5 -----
 core/arch/arm/dts/stm32mp257f-ev1.dts        | 5 -----
 11 files changed, 48 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index ed1320e6e..e85ca1cf4 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -613,10 +613,6 @@
 	clock-error-detect;
 };
 
-&rtc {
-	status = "okay";
-};
-
 &saes {
 	status = "okay";
 };
@@ -665,7 +661,6 @@
 };
 
 &tamp {
-	status = "okay";
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
 	wakeup-source;
diff --git a/core/arch/arm/dts/stm32mp157a-ed1.dts b/core/arch/arm/dts/stm32mp157a-ed1.dts
index 86ad9a06f..457b2f51f 100644
--- a/core/arch/arm/dts/stm32mp157a-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157a-ed1.dts
@@ -370,10 +370,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &timers15 {
 	status = "okay";
 
diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index f35e2cf97..098a391f4 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -529,10 +529,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &sdmmc1 {
 	pinctrl-names = "default", "opendrain", "sleep";
 	pinctrl-0 = <&sdmmc1_b4_pins_a &sdmmc1_dir_pins_a>;
diff --git a/core/arch/arm/dts/stm32mp157d-ed1.dts b/core/arch/arm/dts/stm32mp157d-ed1.dts
index b641b0f4e..5b022baf9 100644
--- a/core/arch/arm/dts/stm32mp157d-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157d-ed1.dts
@@ -369,10 +369,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &timers15 {
 	status = "okay";
 
diff --git a/core/arch/arm/dts/stm32mp157f-ed1.dts b/core/arch/arm/dts/stm32mp157f-ed1.dts
index 3e3a517cd..b2e8cf286 100644
--- a/core/arch/arm/dts/stm32mp157f-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157f-ed1.dts
@@ -370,10 +370,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &timers15 {
 	status = "okay";
 
diff --git a/core/arch/arm/dts/stm32mp15xx-dhcom-som.dtsi b/core/arch/arm/dts/stm32mp15xx-dhcom-som.dtsi
index 57ff02750..79a48ff9f 100644
--- a/core/arch/arm/dts/stm32mp15xx-dhcom-som.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dhcom-som.dtsi
@@ -486,10 +486,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &sdmmc1 {
 	pinctrl-names = "default", "opendrain", "sleep", "init";
 	pinctrl-0 = <&sdmmc1_b4_pins_a &sdmmc1_dir_pins_a>;
diff --git a/core/arch/arm/dts/stm32mp15xx-dhcor-som.dtsi b/core/arch/arm/dts/stm32mp15xx-dhcor-som.dtsi
index cb377ac3d..e7fedb8ab 100644
--- a/core/arch/arm/dts/stm32mp15xx-dhcor-som.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dhcor-som.dtsi
@@ -290,7 +290,3 @@
 &rng1 {
 	status = "okay";
 };
-
-&rtc {
-	status = "okay";
-};
diff --git a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
index 4948dd27f..26d4ecd24 100644
--- a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
@@ -666,10 +666,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &sai2 {
 	clocks = <&rcc SAI2>, <&rcc PLL3_Q>, <&rcc PLL3_R>;
 	clock-names = "pclk", "x8k", "x11k";
diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 85d47f1ba..63e1419ca 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -405,10 +405,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &saes {
 	status = "okay";
 };
@@ -449,7 +445,6 @@
 };
 
 &tamp {
-	status = "okay";
 	wakeup-source;
 	wakeup-parent = <&exti2>;
 	st,tamp-passive-precharge = <2>;
diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 2f2cf6064..3c6b3a11f 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -411,10 +411,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &saes {
 	status = "okay";
 };
@@ -455,7 +451,6 @@
 };
 
 &tamp {
-	status = "okay";
 	wakeup-source;
 	wakeup-parent = <&exti2>;
 	st,tamp-passive-precharge = <2>;
diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index daccb4beb..3aec9a8ec 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -446,10 +446,6 @@
 	status = "okay";
 };
 
-&rtc {
-	status = "okay";
-};
-
 &saes {
 	status = "okay";
 };
@@ -482,7 +478,6 @@
 };
 
 &tamp {
-	status = "okay";
 	wakeup-source;
 	wakeup-parent = <&exti2>;
 	st,tamp-passive-precharge = <2>;
-- 
2.47.2

